appium/webdriver/mobilecommand.py

```diff
@@ -30,9 +30,6 @@ class MobileCommand:
     GET_CURRENT_CONTEXT = 'getCurrentContext'
     SWITCH_TO_CONTEXT = 'switchToContext'
 
-    TOUCH_ACTION = 'touchAction'
```

Comment:
This is a breaking change. We need to bump the major client version in order to remove deprecated endpoints

Reply:
Do you mean for users who implement this endpoint by themselves by referencing this constant? 
This client's implementation for this has been removed in the Python client v3 -> v4, so these are leftover then.

Then, I'll bump this major version after this merge.

Reply:
I did not know the client implementation has been already removed. Then it's probably fine without bumping the major

Reply:
Ah, gotcha